### PR TITLE
Fix lint on components/ui files.

### PR DESCRIPTION
**Pull request by richardxia**
Note that this is probably easier to review with whitespace diffs ignored.

This should be the last of the lint fixes for files that are actually used in production. Everything else left in the `.eslintignore` is either the old admin pages or debugging pages.

### CODE REVIEW

File: app/components/ui/LandingPageEligibilityBlock.jsx [hunk 1]
```diff
@@ -1,46 +1,40 @@
...
-                count={eligibility.service_count}
-                url={this.buildBlockUrl(eligibility.name)}
-              />
-            ))
-            }
-          </div>
-          {/* <a className="scroll-button scroll-button-right">&gt;</a> */}
-        </div>
+      </h2>
+      {/* <a className="scroll-button scroll-button-left">&lt;</a> */}
```
Comment (drcaramelsyrup) [inline app/components/ui/LandingPageEligibilityBlock.jsx]: Should the "Properly implement horizontal scroll buttons" comment be preserved / moved here if this other comment is also being preserved?

File: app/components/ui/Accordion/AccordionItem.jsx [hunk 2]
```diff
@@ -2,33 +2,19 @@ import React from 'react';
...
-        </header>
-        <section>{ this.props.children }</section>
-      </div>
-    );
-  }
-}
+const AccordionItem = ({ children, headerRenderer, title }) => (
+  <div>
+    <header>
+      {headerRenderer(title)}
```
Comment (drcaramelsyrup) [inline app/components/ui/Accordion/AccordionItem.jsx]: Should this be null-checked still (or the headerRenderer set as an `isRequired` prop type)?

Comment (richardxia) [reply app/components/ui/Accordion/AccordionItem.jsx]: I thought the same thing at first, but actually it looks like there's a `defaultProp` for this property that renders just the title as a string.

File: app/components/ui/LandingPageEligibilityBlock.jsx (hunk 1 above)
Comment (richardxia) [reply app/components/ui/LandingPageEligibilityBlock.jsx]: Yeah, sorry, I probably got overzealous deleting and moving around code. I'll add the comment back here.

**Pull request merged**